todo_app/data/trello_items.py

```diff
@@ -0,0 +1,97 @@
+from flask import session
+from datetime import datetime
+import requests
+import os
+
+def trello_get_items():
+    Items=[]
+    uri_path = "/1/boards/" + os.getenv("TRELLO_BOARD_ID") +"/lists"
+    querys = {
+        "cards": "open"
+    }
+    response = call_trello_api(uri_path, "GET", querys).json()
+
+    for trello_list in response:
+        if trello_list['name'] == 'To Do':
+            session["todo_list_id"] = trello_list['id']
```

Comment:
I don't think this is really what you want. `session` is for storing data in the user's browser.

If you want to store things like list IDs to avoid fetching them every time, it's still neater to store it server-side. No need for the browser to know about it.

You could wrap these function in a class with some attributes to hold todo_list_id etc.
Or the simplest change would be just having some global variables:
Set global variable like this:
```python
global todo_list_id
todo_list_id = trello_list['id']
```

Use it in another function like a normal variable:
```python
querys = {
        "idList": todo_list_id
    }
```